dace/config.py

```diff
@@ -39,10 +40,11 @@ def temporary_config():
             Config.set("optimizer", "autooptimize", value=True)
             foo()
     """
-    with tempfile.NamedTemporaryFile() as fp:
-        Config.save(fp.name)
+    with tempfile.NamedTemporaryFile(mode='w+t') as fp:
```

Comment:
what does 't' mean in `mode`?

Reply:
It means text mode, on UNIX `\n` is the newline character, but on Windows it is `\r\n` (https://en.wikipedia.org/wiki/Newline#Representation). In that mode `\r\n` is transformed into `\n` it is essentially a compatibility layer, see `help(open)`.
I used it because it preserves old behaviour, but it is probably not needed.